src/Symfony/Component/Process/Tests/AbstractProcessTest.php

```diff
@@ -564,28 +564,6 @@ public function testIsNotSuccessful()
         $this->assertFalse($process->isSuccessful());
     }
 
-    public function testProcessIsNotSignaled()
-    {
-        if ('\\' === DIRECTORY_SEPARATOR) {
-            $this->markTestSkipped('Windows does not support POSIX signals');
-        }
-
-        $process = $this->getProcess(self::$phpBin.' -v');
-        $process->run();
-        $this->assertFalse($process->hasBeenSignaled());
-    }
-
-    public function testProcessWithoutTermSignalIsNotSignaled()
```

Comment:
they are exact duplicates if I did not make any mistake. And I moved the only assertion in the below test.